lib/board.rb

```diff
@@ -49,7 +49,7 @@ def length_match?(ship, coordinates)
   def consecutive?(coordinates)
     ordinal_values = []
     coordinates.each do |coordinate|
-      ordinal_values << coordinate[0].ord + coordinate[1].to_i
+      ordinal_values << coordinate[0].ord + coordinate[1..2].to_i
```

Comment:
@andilovetto I added this so it now accounts for double-digits
however, once we get to double digits, it throws off our board rendering due to spacing issues. This is more of an Iteration 4 type of problem, but if we get to it, we should add a clause to cap columns at 10.

Reply:
wow this is awesome arty!! as always i am impressed with your wizardry